functions/index.js

```diff
@@ -469,6 +469,44 @@ module.exports.reloadsheetdata = functions.https.onRequest(async (req, res) => {
   res.status(200).send(html_snippets);
 });
 
+async function updateSheetWithGeocodes(country) {
+  const client = await getAuthorizedClient();
+  let data = {}
```

Comment:
Is this used?

Reply:
Nope, data isn't used.  Good catch.  Removed.